code/modules/antagonists/gang/gang.dm

```diff
@@ -548,3 +550,184 @@
 	if(mat_container.materials[SSmaterials.GetMaterialRef(/datum/material/gold)] >= 2000) // if theres at least 1 bar of gold left in the silo, they've failed to heist all of it
 		return FALSE
 	return TRUE
+
+/datum/antagonist/gang/irs
+	show_in_antagpanel = TRUE
+	name = "Internal Revenue Service Agent"
+	roundend_category = "IRS Agents"
+	gang_name = "Internal Revenue Service"
+	gang_id = "IRS"
+	acceptable_clothes = list(/obj/item/clothing/suit/irs,
+							/obj/item/clothing/under/costume/irs,
+							/obj/item/clothing/head/irs)
+	free_clothes = list(/obj/item/clothing/suit/irs,
+							/obj/item/clothing/under/costume/irs,
+							/obj/item/clothing/head/irs,
+						/obj/item/toy/crayon/spraycan)
+	gang_objective = "TODO: Add Objective"
+	antag_hud_name = "IRS"
+
+/datum/antagonist/gang/irs/check_gang_objective()
+	return TRUE
+
+/datum/antagonist/gang/osi
+	show_in_antagpanel = TRUE
+	name = "Office of Secret Intelligence Agent"
+	roundend_category = "O.S.I. Agents"
+	gang_name = "Office of Secret Intelligence"
+	gang_id = "OSI"
+	acceptable_clothes = list(/obj/item/clothing/suit/osi,
+							/obj/item/clothing/under/costume/osi,
+							/obj/item/clothing/glasses/sunglasses/osi)
+	free_clothes = list(/obj/item/clothing/suit/osi,
+							/obj/item/clothing/under/costume/osi,
+							/obj/item/clothing/glasses/sunglasses/osi,
+						/obj/item/toy/crayon/spraycan)
+	gang_objective = "TODO: Add Objective"
+	antag_hud_name = "OSI"
+
+/datum/antagonist/gang/osi/check_gang_objective()
+	return TRUE
+
+/datum/antagonist/gang/tmc
+	show_in_antagpanel = TRUE
+	name = "The Lost M.C. Biker"
+	roundend_category = "Lost M.C. Bikers"
+	gang_name = "The Lost M.C."
+	gang_id = "TMC"
+	acceptable_clothes = list(/obj/item/clothing/suit/tmc,
+							/obj/item/clothing/under/costume/tmc,
+							/obj/item/clothing/head/tmc)
+	free_clothes = list(/obj/item/clothing/suit/tmc,
+							/obj/item/clothing/under/costume/tmc,
+							/obj/item/clothing/head/tmc,
+						/obj/item/toy/crayon/spraycan)
+	gang_objective = "TODO: Add Objective"
+	antag_hud_name = "LostMC"
+
+/datum/antagonist/gang/tmc/check_gang_objective()
+	return TRUE
+
+/datum/antagonist/gang/pg
+	show_in_antagpanel = TRUE
+	name = "Powder Ganger"
+	roundend_category = "Powder Gangers"
+	gang_name = "Powder Gangers"
+	gang_id = "PG"
+	acceptable_clothes = list(/obj/item/clothing/suit/pg,
+							/obj/item/clothing/under/costume/pg,
+							/obj/item/clothing/head/pg)
+	free_clothes = list(/obj/item/clothing/suit/pg,
+							/obj/item/clothing/under/costume/pg,
+							/obj/item/clothing/head/pg,
```

Comment:
suggestion: add an IED or two to the list of "free clothes" that powder gangers get

they're cheap enough to make normally and don't actually do that much damage, so it shouldn't be too unbalancing to add

Reply:
this'll be addressed when I get weaponry sprites for gangs

Reply:
we have IEDs as an item already, though

Reply:
Yeah, but what did powder gangers use?

Reply:
any explosives they could get their hands on, no?